cmd/pullrequest-init/main.go

```diff
@@ -19,18 +19,18 @@ import (
 	"context"
 	"flag"
 	"fmt"
-	"os"
-
 	"github.com/tektoncd/pipeline/pkg/pullrequest"
 	"go.uber.org/zap"
 	"knative.dev/pkg/logging"
+	"os"
 )
 
 var (
-	prURL    = flag.String("url", "", "The url of the pull request to initialize.")
-	path     = flag.String("path", "", "Path of directory under which PR will be copied")
-	mode     = flag.String("mode", "download", "Whether to operate in download or upload mode")
-	provider = flag.String("provider", "", "The SCM provider to use. Optional")
+	prURL     = flag.String("url", "", "The url of the pull request to initialize.")
+	path      = flag.String("path", "", "Path of directory under which PR will be copied")
+	mode      = flag.String("mode", "download", "Whether to operate in download or upload mode")
+	provider  = flag.String("provider", "", "The SCM provider to use. Optional")
+	sslVerify = flag.Bool("sslVerify", true, "Enable/Disable SSL verification in the git client. Defaults to true")
```

Comment:
I'd like to propose the flag be renamed to `-insecure-skip-ssl-verify` to make things painstakingly clear.  As an added benefit this would also make the flag `false` by default, which matches Go's default value for bools.  The reason I prefer that is that if this code were ever refactored and the default value accidentally dropped, the binary would still be "secure by default", barring some malicious attempt to set it differently.

Reply:
I can make those changes (although likely not until Monday) ... the only reason I called it sslVerify was to be consistent across pipelineresources .... for type git I introduced sslVerify as an option (but this is essentially the option name in git config that it alludes to) .... What I might do,  is change your suggested name slightly and go with `insecure-skip-tls-verify`

Reply:
Should be good to re-review now